examples/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
-cmake_minimum_required(VERSION 2.8)
```

Comment:
Are these changes on purpose or did they sneak in?

Reply:
That is on purpose. `cmake_minimum_required(VERSION)` has side-effects (sigh) such as setting all `CMP00XX` policies to `NEW` if they were defined in cmake version `VERSION` or to unset the variable otherwise. It should only be called once.

Reply:
🤦‍♂️